### PR TITLE
refs:Task #11342 init project

### PR DESCRIPTION
**Pull request by MinhHoang2003**
# Issue name
- init project
# Issue link
- https://edu-redmine.sun-asterisk.vn/issues/11342
# Screenshots or screen cast
-

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.colornotetaking"
```
Comment (DoanVanToan) [inline app/build.gradle]:
```suggestion
        applicationId "com.sun.colornotetaking"
```

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: remove hardcode
tất cả string show với user cần được define trong String.xml

**Review by DoanVanToan — COMMENTED** [review]
Change package and app id to
com.sun. ...

**Comment by MinhHoang2003** [comment]
ready

**Pull request merged**